index.js

```diff
@@ -25,12 +25,34 @@ const panel = require('sdk/panel').Panel({
   }
 });
 
-getActiveView(panel).setAttribute('noautohide', true);
```

Comment:
'noautohide' needs to live outside of the `makePanelDraggable` call to ensure the panel doesn't disappear on click for linux users